README.md

```diff
@@ -90,6 +90,8 @@ Alternatively if indexer is running on the same host as the archival node, a sim
 ~$ algorand-indexer daemon --algod-net yournode.com:1234 -d /path/to/algod/data/dir --postgres "user=readonly password=YourPasswordHere {other connection string options for your database}"
 ```
 
+**Note**: Keeping indexer daemon as close as possible to database helps minimize latency. For example if using AWS EC2 and RDS, we suggest putting EC2 in the same VPC, Region, and even Availability Zone.
```

Comment:
Could you move this to above the command examples? I think it would fit nicely between the `In this mode, ...` and `You s should use a process manager...` sections. No need for `**Note**: `, everything here is important to read not just this recommendation.

Reply:
done